midi_app_controller/gui/binds_editor.py

```diff
@@ -227,7 +225,7 @@ class BindsEditor(QDialog):
 
     Attributes
     ----------
-    save_binds : Callable[[List[KnobBind], List[ButtonBind]], None]
+    save_binds : callable[[list[KnobBind], list[ButtonBind]], None]
```

Comment:
there is no callable[...] generic!! You still have to do `from typing import Callable` and use `Callable`.

See e.g. https://stackoverflow.com/questions/66902082/did-python-3-9-update-how-to-type-hint-the-function-type